docs/source/reference/modding/settings/game.rst

```diff
@@ -51,6 +51,18 @@ Whether or not the chance of success will be displayed in the enchanting menu.
 
 The default value is false. This setting can only be configured by editing the settings configuration file.
 
+advanced pickpocketing
+----------------------
+
+:Type:		boolean
+:Range:		True/False
+:Default:	False
+
+If this setting is true, an engine will use an item weight instead of value to calculate chance of successful pickpocketing.
+Also a "reverse pickpocketing" feature will be enabled in this case.
```

Comment:
"**the** engine", "**the** item weight"/"**the** weight of an item", "**the** value", "**the** chance" and "**the** <...> feature" on this and previous lines. Definite article.